dashboards/grafana/configmap/grafana-dashboard-keptn-overview.yaml

```diff
@@ -33,8 +33,8 @@ data:
       "panels": [
```

Comment:
i guess this configmap also needs the `grafana_dashboard` label?

Reply:
good catch apparently my regex failed t find it 😄 I will check again all